dts/arm/nxp/nxp_rt11xx.dtsi

```diff
@@ -1062,6 +1062,54 @@
 			status = "okay";
 		};
 
+
+		qdec1: qdec@40174000 {
+			compatible = "nxp,mcux-qdec";
+			reg = <0x40174000 0x4000>;
+			interrupts = <165 0>;
+			status = "disabled";
+		};
+
+		qdec2: qdec@40178000 {
+			compatible = "nxp,mcux-qdec";
+			reg = <0x40178000 0x4000>;
+			interrupts = <166 0>;
+			status = "disabled";
+		};
+
+		qdec3: qdec@4017c000 {
+			compatible = "nxp,mcux-qdec";
+			reg = <0x4017c000 0x4000>;
+			interrupts = <167 0>;
+			status = "disabled";
+		};
+
+		qdec4: qdec@40180000 {
```

Comment:
should we call these nodes qdc instead of qdec to match the RT11XX RM, to avoid confusion? I am actually not sure why our binding either is called qdec since that is not the name of the peripheral, but that doesn't need to be addressed by this PR